samples/bot-formatting-cards/nodejs/teamsapp.yml

```diff
@@ -3,4 +3,4 @@
 # Visit https://aka.ms/teamsfx-actions for details on actions
 version: 1.0.0
 
-environmentFolderPath: ./env
+environmentFolderPath: ./env
```

Comment:
bot-formatting-cards only has this change?

Reply:
Files we pushed already
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/9b5e3982-e7bb-4a55-8246-73e3a60bcab5)

Reply:
fixed